packages/apm/src/integrations/express.ts

```diff
@@ -56,42 +56,66 @@ export class Express implements Integration {
  * // error handler
  * app.use(function (err, req, res, next) { ... })
  */
-function wrap(fn: Function, getCurrentHub: () => Hub): RequestHandler | ErrorRequestHandler {
+function wrap(fn: Function): RequestHandler | ErrorRequestHandler {
   const arrity = fn.length;
 
   switch (arrity) {
     case 2: {
       return function(this: NodeJS.Global, _req: Request, res: Response): any {
-        const span = getCurrentHub().startSpan({
-          description: fn.name,
-          op: 'middleware',
-        });
-        res.once('finish', () => span.finish());
+        // tslint:disable: no-unsafe-any
```

Comment:
Can we maybe use inline `// tslint:disable-line:no-unsafe-any` instead? We can miss something easily with enable/disable.

Reply:
+ all other places with same issue.

Reply:
Used union type to fix this `any`